api/src/main/java/org/openmrs/module/fhir/api/AllergyIntoleranceService.java

```diff
@@ -55,8 +55,31 @@
 	 * Ger allergies by peron uuid
 	 *
 	 * @param personId uuid
-	 * @return
+	 * @return fhir allergy resource list
 	 */
 	List<AllergyIntolerance> searchAllergiesByPatientUuid(String personId);
 
+	/**
+	 * Create allergy tolerance
+	 *
+	 * @param allergyIntolerance to be created
+	 * @return created allergy tolerance obj
```

Comment:
TYPO? I believe there should be "intolerance".

Reply:
TYPO, thanks